terraso_backend/apps/soil_id/models/depth_dependent_soil_data.py

```diff
@@ -23,3 +23,16 @@ class DepthDependentSoilData(BaseModel):
     soil_data_input = models.ForeignKey(SoilData, on_delete=models.CASCADE)
     depth_top = models.IntegerField(blank=True)
     depth_bottom = models.IntegerField(blank=True)
+    slope = models.IntegerField(blank=True)
```

Comment:
for slope: 
- this one is not depth dependent, so it should be a field on `soil_data`
- for this and other fields with valid value ranges, we can use django's `MaxValueValidator` and `MinValueValidator` to ensure the percentage value is between 0 and 100: https://docs.djangoproject.com/en/4.2/ref/validators/#maxlengthvalidator
- we also need enum fields for slope shape
- for all the fields, we need to specify `null=True` (blank means that when users submit data they can omit the field, null means that the field is allowed to be null in the database. so if we had `blank=True` and `null=False`, we would need to supply a default value. but in this case we just want to allow nulls). this is true for all of the fields we're adding in this PR